model_analyzer/triton/server/server_docker.py

```diff
@@ -27,11 +27,12 @@ class TritonServerDocker(TritonServer):
     Concrete Implementation of TritonServer interface that runs
     triton in a docker container.
     """
-
-    def __init__(self, model_path, image, config):
+    def __init__(self, gpus, model_path, image, config):
```

Comment:
Not sure how much this matters but could we place gpus as the last arguments? I think arguments order should be a bit intuitive.